cohorts/utils.py

```diff
@@ -17,6 +17,25 @@
 from collections import namedtuple
 import sys
 import logging
+from os import path
+
+def get_cache_dir(cache_dir, cache_root_dir=None, *args, **kwargs):
+    """
+    Return full cache_dir, according to following logic:
+        - if cache_dir is a full path (per path.isabs), return that value
+        - if not and if cache_root_dir is not None, join two paths
+        - otherwise, log warnings and return None
+    Separately, if args or kwargs are given, format cache_dir using kwargs
+    """
+    cache_dir = cache_dir.format(*args, **kwargs)
+    if path.isabs(cache_dir):
```

Comment:
This logic feels a little incomplete to me. What if an absolute path is passed in along with a cache_root_dir? Can we have a warning for that scenario?